sovtokenfees/sovtokenfees/test/conftest.py

```diff
@@ -4,8 +4,12 @@
 from sovtokenfees.main import integrate_plugin_in_node as enable_fees
 
 # fixtures, do not remove
-from plenum.test.conftest import *
+from indy_node.test.conftest import *
```

Comment:
Can you specify what do you actually want to import? "import *" is not a 'clean' statement.

Reply:
When I remove this statement I see that there are many dependencies on conftests. I believe the wildcard is easier here than importing every single dependency